### DISCUSSION TITLE
Rate limited from the youtube API based on IP?

### DISCUSSION THREAD

**Discussion by dmitsuki** [TeamNewPipe/NewPipe#11660] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Does youtube ban IP's based on using the download API?

### Additional information

I am currently unable to search for videos or view comments, but I am able to watch videos. If I access youtube through a browser, with no adblock, the same problem persist. The issue goes away if I use a VPN. Is this possibly a rate ban, and has anything experienced anything similar?

**Comment by txsetanta** [comment]
same here, since 2 days I have to use  vpn on every device around the house, to use newpipe, it seems that single connection per IP is only allowed,

**Comment by AudricV** [comment]
We have no answer right now for your question. Closing in favor of #11661.
